File: Python/tests/test_bandit4arm_2par_lapse.py
import pytest

from hbayesdm.models import bandit4arm_2par_lapse


def test_bandit4arm_2par_lapse():
    _ = bandit4arm_2par_lapse(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_bandit4arm_2par_lapse.R
context("Test bandit4arm_2par_lapse")
library(hBayesDM)

test_that("Test bandit4arm_2par_lapse", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bandit4arm_2par_lapse(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
